src/components/index/IndexHeader.tsx

```diff
@@ -71,15 +71,15 @@ function Stats (): JSX.Element {
         <ReactNumberFormat
           displayType='text'
           thousandSeparator
-          value={stats.price.usd}
+          value={stats?.price?.usd}
```

Comment:
hmm, this is not throwing an error on my end and checking the types, it should be fine without the optional chaining

Reply:
As discussed, optional chaining will be required as a safety net against undefined data retrieved from the api